### DISCUSSION TITLE
RE8 crashes on startup.

### DISCUSSION THREAD

**Discussion by ivankd2** [praydog/REFramework#560] [General Questions] [answered]
Greetings! Thank you for the new sensations when passing these games! 
I have a WMR headset. I played RE 2,3 and 7 parts of the game with your VR mod. Part 8 can't start. There are no critical errors in re2_framework_log.txt. The program just closes and the game crashes along with the program. What could be the reason? Tried deleting openvr_api.dll . Same situation. RE 2,3,7 work well with and without openxr_loader.dll. 
[re2_framework_log.txt](https://github.com/praydog/REFramework/files/9896779/re2_framework_log.txt)

**Comment by praydog** [comment, marked as answer]
Pirated versions of RE8 are not supported.

**Reply by ivankd2** [reply]
I understand. Thank you ! )

**Reply by haseebb420** [reply]
> Pirated versions of RE8 are not supported.

Will it work if I have the xbox PC gamepass version of RE8?
